dependency-check-suppressions.xml

```diff
@@ -144,4 +144,11 @@
         <gav regex="true">.*</gav>
         <cve>CVE-2018-1000840</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+			Not applicable - Randomness vulnerability when using SecureRandomFactoryBean
+			]]></notes>
+        <gav regex="true">.*</gav>
+        <cve>CVE-2019-3795</cve>
```

Comment:
How about upgrade the affected versions of jars rather than ignoring here?
See details here: https://pivotal.io/security/cve-2019-3795